plane/src/database/subscribe.rs

```diff
@@ -352,11 +352,11 @@ impl EventSubscriptionManager {
     }
 }
 
-pub async fn emit_impl<'c, T, E>(db: E, key: Option<&str>, payload: &T) -> Result<(), sqlx::Error>
-where
-    T: NotificationPayload,
-    E: PgExecutor<'c>,
-{
+pub async fn emit_impl<T: NotificationPayload>(
```

Comment:
These changes were made following [this comment](https://github.com/launchbadge/sqlx/discussions/1136#discussioncomment-566896) about using sqlx.